charts/evm-rollup/templates/configmap.yaml

```diff
@@ -5,13 +5,12 @@ metadata:
   namespace: {{ include "rollup.namespace" . }}
 data:
   ASTRIA_CONDUCTOR_LOG: "astria_conductor={{ .Values.config.logLevel }}"
-  {{- if (index .Values "celestia-node").enabled }}
-  TOKEN_SERVER_URL: "{{ include "celestiaNode.service.addresses.token" (index .Subcharts "celestia-node") }}"
-  ASTRIA_CONDUCTOR_CELESTIA_NODE_HTTP_URL: "{{ include "celestiaNode.service.addresses.rpc" (index .Subcharts "celestia-node") }}"
-  {{- else }}
+  {{- if .Values.config.celestia.node.enabled }}
   TOKEN_SERVER_URL: "{{ .Values.config.celestia.token }}"
-  ASTRIA_CONDUCTOR_CELESTIA_NODE_HTTP_URL: "{{ .Values.config.celestia.rpc }}"
+  {{- else }}
+  ASTRIA_CONDUCTOR_CELESTIA_BEARER_TOKEN: "{{ .Values.config.celestia.token }}"
   {{- end }}
```

Comment:
When deploying a celestia node to read from, fetch celestia auth token through the token server. Otherwise, use a raw value